rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -374,7 +378,7 @@ fn config_path() -> PathBuf {
     }
     cfg_dir
 }
-fn data_path() -> PathBuf {
+pub fn data_path() -> PathBuf {
```

Comment:
Doesn't need to be pub for a submodule to access it. Submodules can always access private items of its supermodules.